docs/support.md

```diff
@@ -19,9 +19,9 @@ If you are using Xen Orchestra via the sources, you can ask questions and try to
 * In our [forum](https://xcp-ng.org/forum/category/12/xen-orchestra)
 * In our IRC - `#xen-orchestra` on `Freenode`
 
-However, there's no guarantee you will receive an answer and no guaranteed response time. If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
+If you are using XO from sources, we encourage you to give back to the community by assisting other users via these two avenues as well.  
 
-If you are using Xen Orchestra in production, please subscribe to a plan.
+Lastly while Xen Orchestra is free to use and change, supporting and developing it takes a lot of effort. If you consider support business critical please sign-up for one of our [support plans](https://xen-orchestra.com/#!/xo-pricing).
```

Comment:
If you want more details than the original sentence, what do you think about:

"If you are using Xen Orchestra in production, please subscribe to a plan to get turnkey XO virtual Appliance (XOA) with QA, web updater and pro support. Xen Orchestra itself is Open Source, but supporting, developing  and maintaining it takes a lot of effort."

Reply:
That sounds a little pushy to get people to purchase the XOA. And while I agree, production use should mandate that a business need support, I can't make that justification for every business. 

A not-for-profit charity may want to use the solution but be scared away from the product by seeing wording like that.

Reply:
Proposed changes have been made